src/me/drton/flightplot/export/AbstractTrackExporter.java

```diff
@@ -50,6 +54,7 @@ private void feedAnalyzers(TrackPoint point) {
 
     public void exportToFile(File file, String title) throws IOException {
         this.writer = initWriter(file, title);
+        altOffset = configuration.getAltOffset();
```

Comment:
I would just access this field over the configuration directly when used, instead of a member that is "by chance" initialized at the right time ;)